checklists/alz_checklist.en.json

```diff
@@ -945,27 +928,26 @@
             "service": "VNet",
             "guid": "b1c82a3f-2320-4dfa-8972-7ae4823c8930",
             "id": "D05.08",
-            "ammp": true,
             "severity": "High",
             "training": "https://learn.microsoft.com/learn/paths/secure-networking-infrastructure/",
             "link": "https://learn.microsoft.com/azure/ddos-protection/ddos-protection-reference-architectures"
         },
         {
             "category": "Network Topology and Connectivity",
             "subcategory": "Hybrid",
-            "text": "Ensure that you have investigated the possibility to use ExpressRoute as primary connection to Azure.",
+            "text": "Determine if ExpressRoute should be used as the primary connection to Azure.",
```

Comment:
Are there any reasons why customers shouldn't use ER as primary path to access Azure?

Reply:
Cost and complexity mostly - do you think we should remove this?  It's a weak recommendation because it falls into the "consider" but it is something that is high visibility.